[2023-04-10 16:47:58,056][moc][INFO][color_logger.py:203] Logfile: /home/chenyiqi/wafer_scale_chip_dse_framework/logs/moc_2023-04-10-16-47-58-055835.log
[2023-04-10 16:47:58,057][moc][WARNING][generic_smbo.py:387] Exception when calling objective function: local variable '_history' referenced before assignment
config: Configuration(values={
  'fidelity': 2,
  'var00': 128,
  'var01': 1024,
  'var02': 32,
  'var03': 64,
  'var04': 4,
  'var05': 4,
  'var06': 1,
  'var07': 48,
  'var08': 32,
  'var09': 900,
  'var10': 6,
  'var11': 11,
})

[2023-04-10 16:47:58,057][moc][INFO][generic_smbo.py:418] Iter 16, objectives: [inf].
[2023-04-10 16:48:02,316][moc][WARNING][util_funcs.py:231] Keyword argument "num_objs" is deprecated and will be removed in a future version. Please use "num_objectives" instead.
